pycroft/lib/membership.py

```diff
@@ -162,6 +162,22 @@ def remove_member_of(
                    user=user, author=processor)
 
 
+@with_transaction
```

Comment:
Remove the `@with_transaction` please, this is an ancient relic – and not necessary, since you correctly took care to `session.commit()` in the `web/` endpoint which consumes this function.

Reply:
Removed